packages/react-core/src/demos/examples/Tabs/GrayTabs.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import {
+  PageSection,
+  PageSectionVariants,
+  Tabs,
+  Tab,
+  TabContent,
+  TabContentBody,
+  TabTitleText,
+  Title,
+  Grid,
+  GridItem,
+  Flex,
+  FlexItem,
+  Card,
+  CardBody,
+  CardHeader
+} from '@patternfly/react-core';
+import DashboardWrapper from '../../examples/DashboardWrapper';
+
+export const GrayTabsDemo: React.FunctionComponent = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState(0);
+  const [activeNestedTabKey, setActiveNestedTabKey] = React.useState(10);
+
+  // Toggle currently active tab
+  const handleTabClick = (_: React.MouseEvent, tabIndex: string | number) => {
+    if (typeof tabIndex === 'string') {
```

Comment:
It looks like all your keys are numbers. Does it work if you make the type of `tabIndex` a number?